pontoon/messaging/emails.py

```diff
@@ -310,4 +313,259 @@ def send_notification_digest(frequency="Daily"):
+        )
+        msg.attach_alternative(body_html, "text/html")
+        msg.send()
+
+    pks = users.values_list("pk", flat=True)
+    UserProfile.objects.filter(user__in=pks).update(onboarding_email_status=3)
+
+    log.info(f"3rd onboarding emails sent to { len(users) } users.")
+
+
+def send_inactive_contributor_emails(users):
+    """
+    Sends an email to an inactive contributor.
+    """
+    log.info("Start sending inactive contributor emails.")
+
+    email_content = EmailContent.objects.get(email="inactive_contributor")
+    content = jinja_env.from_string(email_content.body).render()
+
+    subject = email_content.subject
+    template = get_template("messaging/emails/transactional.html")
+
+    body_html = template.render(
+        {
+            "content": content,
+            "subject": subject,
+        }
+    )
+    body_text = html_to_plain_text_with_links(body_html)
+
+    for user in users:
+        msg = EmailMultiAlternatives(
+            subject=subject,
+            body=body_text,
+            from_email=settings.DEFAULT_FROM_EMAIL,
+            to=[user.contact_email],
+        )
+        msg.attach_alternative(body_html, "text/html")
+        msg.send()
+
+    pks = users.values_list("pk", flat=True)
+    now = timezone.now()
+    UserProfile.objects.filter(user__in=pks).update(last_inactive_reminder_sent=now)
+
+    log.info(f"Inactive contributor emails sent to { len(users) } users.")
+
+
+def send_inactive_translator_emails(users, translator_map):
+    """
+    Sends an email to an inactive translator.
+    """
+    log.info("Start sending inactive translator emails.")
+
+    email_content = EmailContent.objects.get(email="inactive_translator")
+    subject = email_content.subject
+    template = get_template("messaging/emails/transactional.html")
+
+    for user in users:
+        try:
+            locale = list(translator_map[user.pk])[0]
+        except IndexError:
+            log.error(f"User { user } is not a translator of any locale.")
+            continue
+
+        content = jinja_env.from_string(email_content.body).render({"locale": locale})
+        body_html = template.render(
+            {
+                "content": content,
+                "subject": subject,
+            }
+        )
+        body_text = html_to_plain_text_with_links(body_html)
+
+        msg = EmailMultiAlternatives(
+            subject=subject,
+            body=body_text,
+            from_email=settings.DEFAULT_FROM_EMAIL,
+            to=[user.contact_email],
+        )
+        msg.attach_alternative(body_html, "text/html")
+        msg.send()
+
+    pks = users.values_list("pk", flat=True)
+    now = timezone.now()
+    UserProfile.objects.filter(user__in=pks).update(last_inactive_reminder_sent=now)
+
+    log.info(f"Inactive translator emails sent to { len(users) } users.")
+
+
+def send_inactive_manager_emails(users, manager_map):
+    """
+    Sends an email to an inactive manager.
+    """
```

Comment:
Is a manager becoming inactive significant enough that someone else should also be notified about it?

Reply:
Adding @bcolsson for visibility and a potential followup.

Reply:
Since these don't repeat after the first instance of it being sent, I'm not sure how useful it would be. Since we have our locale health dashboard and will be looking at those more proactively going forward, I don't think we need to add anything here.